systemvm/debian/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -194,7 +194,7 @@ def _redundant_on(self):
         heartbeat_cron.commit()
 
         proc = CsProcess(['/usr/sbin/keepalived'])
```

Comment:
@ustcweizhou specifying "/usr/sbin/keepalived" here may not be required as grep is used instead.

I understand this issue is with _keepalived_ process, but on a broader view, other processes might have similar issue later, can this be fixed in _CsProcess.py find()_ ?

Reply:
@sureshanaparti yes, of course.
To be sure that is not regression issue, I choosed a easier way to fix it.

Reply:
ok @weizhouapache , got it. Note that, if the same issue repeats for other processes later, then _CsProcess.py_ have to be fixed.